src/forms/components/UserForm/index.js

```diff
@@ -0,0 +1,88 @@
+import React, {useState, Fragment} from 'react';
+import {Text, TouchableOpacity} from 'react-native';
+import {Field, reduxForm} from 'redux-form';
+import {useDispatch} from 'react-redux';
+
+import {createFields} from '../../utils';
+import {USER_FORM} from '../../constants';
+
+import actionsCreator from '../../../redux/auth/actions';
+import {required, email, minChar, checkPass} from '../../validate';
+
+import {styles} from './style';
+
+const UserForm = ({navigation, ...props}) => {
+  const [changeForm, setChangeForm] = useState(true);
+  const dispatch = useDispatch();
+
+  const {handleSubmit, reset, submitting} = props;
```

Comment:
Don't destructurate props like this. Just write those props you need in the function param.